app/views/tag/_topicCard.html.erb

```diff
@@ -32,7 +32,7 @@
       </div>
 
       <div class="card-footer" style="background-color: inherit; border:none;">
-        <a style="padding-top:15px;text-decoration:underline;color:#808080;display:inline-block;" href="/tag/<%= tag.name %>"><%= tag.count %> <%= translation('tag.index.posts') %> &raquo;</a>
+        <a style="padding-top:15px;text-decoration:underline;color:#808080;display:inline-block;" href="/tag/<%= tag.name %>"><%= Tag.tagged_node_count(tag.name) %> <%= translation('tag.index.posts') %> &raquo;</a>
```

Comment:
I'm curious about the difference between `tag.count` and `Tag.tagged_node_count(tag.name)` does it mean that tag.count only shows the total number of posts including wikis and `Tag.tagged_node_count(tag.name)` only shows the number of posts?

Is it related to why the classroom-community-science post was showing 0 posts when there was 1?

Reply:
It is related but I think the main difference is that .count is stored in the database and has to be recalculated to be accurate, while the other is dynamically recalculated each time, which is less efficient but more accurate (as long as there are any possible ways to miss recalculating!). I think this could be refactored or renamed for clarity but it isn't a huge deal.